### PR TITLE
Spliting interface

### PR DESCRIPTION
**Pull request by TomekPaluch**

### CODE REVIEW

File: django_react_proj/users-fe/src/components/Home/Home.js [hunk 1]
```diff
@@ -22,7 +39,30 @@ const Home = () => {
     <Container className={styles.Container}>
       <Row>
         <Col>
-          <UserList users={users} resetState={getUsers} />
+          <Button type="button" onClick={() => setCurrentDate(getPreviousDay)}>
+            Poprzedni dzień
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/Home/Home.js:43]: English or Polish?

**Review by bakowroc — APPROVED** [review]

File: django_react_proj/users-fe/src/components/Home/Home.js (hunk 1 above)
Comment (TomekPaluch) [reply django_react_proj/users-fe/src/components/Home/Home.js:43]: It should be polish

File: django_react_proj/users-fe/package.json [hunk 2]
```diff
@@ -18,6 +18,8 @@
     "moment": "^2.29.1",
```
Comment (karolszymonczyk) [inline django_react_proj/users-fe/package.json:17]: iirc you don't use`moment` anymore. Remove this dependency.

File: django_react_proj/users-fe/src/components/Home/Home.js [hunk 3]
```diff
@@ -1,19 +1,36 @@
...
   const [users, setUsers] = useState();
-
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [modalFlag, setModalFlag] = useState(false);
+  const toggle = () => {
+    setModalFlag((previous) => !previous);
+  };
+  const button = <Button onClick={toggle}>Kalendarz</Button>;
   const getUsers = () => {
     axios.get(API_URL).then((res) => setUsers(res.data));
```
Comment (karolszymonczyk) [inline django_react_proj/users-fe/src/components/Home/Home.js:32]: what about using `async/await`? And maybe try to catch some errors if request fail

Comment (TomekPaluch) [reply django_react_proj/users-fe/src/components/Home/Home.js:32]: It has been fixed on another branch

File: django_react_proj/users-fe/package.json (hunk 2 above)
Comment (TomekPaluch) [reply django_react_proj/users-fe/package.json:17]: I will do it

**Pull request merged**